maestro/providers/aws/vpc_location.py

```diff
@@ -33,6 +33,12 @@ def get_vpc_id(vpc_name):
     '''
     filters = [{'Name': 'tag:Name', 'Values': ['%s' % vpc_name]}]
     vpcs = list(ec2.vpcs.filter(Filters=filters))
+
+    if not vpcs:
+        print(color.RED + "Couldn't find the ID for your vpc, vpc_name: " + vpc_name + ", check the name and try again"
```

Comment:
It's generally preferred to use string formatting rather than `+`:

```
print(color.RED + "Couldn't find the ID for your vpc, vpc_name: {}, check the name and try again".format(vpc_name) + color.END)
```

Reply:
yeap I agree.. but all the existing code used + so I was trying to keep the homogeneity..

Reply:
will push a commit

Reply:
Ah, ok. In that case you could go either way. Looking through more of the code, I think we could spend some time doing a cleanup pass to make this repo more idiomatic.

Reply:
yeah since max is gone definitely need someone to own this.

Reply:
Yeah, I'll write a DEVOPS ticket and assign it to myself. Happy to take the lead on that.

Reply:
I need to make a job in teamcity to build the docker image for this automatically after pushing to master. i'll take care of that piece.

Reply:
I can help too.. We are starting to rely on this codebase a lot from data engg side.